src/renderer/page/search/view.jsx

```diff
@@ -48,52 +48,54 @@ class SearchPage extends React.PureComponent<Props> {
 
     return (
       <Page noPadding>
-        {query &&
-          isValid && (
-            <div className="search__top">
-              <div className="file-list__header">
+        <section className="search">
+          {query && isValid && (
+            <header className="search__header">
+              <h1 className="search__title">
                 {`lbry://${query}`}
                 <ToolTip
                   icon
                   body={__('This is the resolution of a LBRY URL and not controlled by LBRY Inc.')}
                 >
                   <Icon icon={ICONS.HELP} />
                 </ToolTip>
-              </div>
+              </h1>
               {isChannel ? (
                 <ChannelTile size="large" uri={uri} />
               ) : (
                 <FileTile size="large" displayHiddenMessage uri={uri} />
```

Comment:
The placeholder seems broken for this top result on the search page. For channels it looks fine.

Reply:
I must have a great connection because I don't see these hitches. I'm adding min-heights to things now.

Reply:
Try clearing cache then try again. It might be already fetched. Or just search for another channel.